sql-db/hibernate-reactive/src/test/java/io/quarkus/ts/reactive/MariaDbDatabaseIT.java

```diff
@@ -0,0 +1,36 @@
+package io.quarkus.ts.reactive;
+
+import io.quarkus.test.bootstrap.DefaultService;
+import io.quarkus.test.bootstrap.RestService;
+import io.quarkus.test.scenarios.QuarkusScenario;
+import io.quarkus.test.services.Container;
+import io.quarkus.test.services.QuarkusApplication;
+
+@QuarkusScenario
+public class MariaDbDatabaseIT extends AbstractReactiveDatabaseIT {
+
+    private static final String MYSQL_USER = "quarkus_test";
+    private static final String MYSQL_PASSWORD = "quarkus_test";
+    private static final String MYSQL_DATABASE = "quarkus_test";
+    private static final int MYSQL_PORT = 3306;
+
+    @Container(image = "${mariadb.102.image}", port = MYSQL_PORT, expectedLog = "Only MySQL server logs after this point")
+    static DefaultService database = new DefaultService()
```

Comment:
Please instead of `DefaultService` use `MariaDbService`

Reply:
I have doubts about that, since Quarkus Hibernate doesn't yet support maria db(see comment on line 28). We will need to override parameters anyway, so I suggest we will do it later, when we have a proper support. WDYT?

Reply:
Then add a //TODO

Reply:
"Quarkus Hibernate doesn't yet support maria db" what is not supported specifically? + to add a mention to include OpenShiftMariaDb scenario once resolved with TODO comment

Reply:
TODO added